Domain/Enums/TimeOffType.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Domain.Enums
+{
+    public enum TimeOffType : int
```

Comment:
lets try order them and isPaid factor. So order would be following:
ForceMajeureAdministrativeLeave = 1,
AdministrativeUnpaidLeave = 2,
SocialLeave = 3,
SickLeaveWithoutDocuments = 4
SickLeaveWithDocuments= 5,
StudyLeave = 6,
PaidLeave = 7